plex/api.py

```diff
@@ -386,6 +391,8 @@ def getPlexItemAsListItem(
         if not plexItemId:
             raise ValueError('invalid plexItemId')
 
+        # TODO: How do we get the importSettings and providerSettings here?
```

Comment:
Is this still valid? Maybe `allowDirectPlay` should be passed into `getPlexItemAsListItem()`?

Reply:
Yep, that isn't needed anymore. Got allowDirectPlay into that function and forgot to finish passing it through.